test/unit/data/custom-metadata-json/ambiguous_scenarios_specified_v1.json

```diff
@@ -0,0 +1,100 @@
+[
```

Comment:
Hi @LennDG 

Tnx for the changes. I only think you missed something. If I now generate the files there is no `custom-metadata` in the report, there is also no custom metadata in the pushed JSON files

Reply:
I overwrite the metadata of the JSON files and have not added a `custom-metadata` field. This is to use the existing code as much as possible without needing changes.